ml-agents-envs/mlagents_envs/base_env.py

```diff
@@ -247,90 +246,54 @@ def empty(spec: "BehaviorSpec") -> "TerminalSteps":
 class ActionType(Enum):
     DISCRETE = 0
     CONTINUOUS = 1
+    HYBRID = 2
```

Comment:
Not used? I think we decided against it in the design review, right?

Reply:
Removed. In reality I could remove ActionType throughout the whole codebase (as per @dongruoping 's comment below). Thoughts? @vincentpierre @ervteng